src/store.js

```diff
@@ -78,6 +80,17 @@ export class Store {
     }
   }
 
+  applyMixins (options) {
```

Comment:
It would more appropriate to put this logic in https://github.com/vuejs/vuex/blob/dev/src/module/module.js. Because `Module` class is responsible for each module.